sonic_api.yaml

```diff
@@ -2346,14 +2346,14 @@ definitions:
       nexthop:
         type: string
         description: >-
-          ip address of nexthop.
+          ip address of nexthop. 'comma' seperated list for ecmp (equal cost multipath) routes; User can optionally specify emtpy '' strings for positional arguments.
           When nexthop_type is 'ip', then nexthop is the CE IP address;
           When nexthop_type is 'vxlan_tunnel', the next hop is the PA address of the customer VM
       mac_address:
         type: string
         description: mac_address of the target VM
       ifname:
-        type: string
+        type: string. 'comma' seperated list for ecmp routes; User can optionally specify empty '' strings for positional arguments.
```

Comment:
The description needs to move down in the description field.

Reply:
Thanks, done